libs/directory.js

```diff
@@ -0,0 +1,29 @@
+import fs from 'fs';
+import path from 'path';
+import { toReadFile } from './file.js';
+
+export const toReadDirectory = (pathFromCli) => {
+  let filesMD = []
+  fs.readdir(pathFromCli, function callback(err, list) {
+    if (err) return console.error(err)
+
+    list.forEach(file => {
+     
+      if (path.extname(file) == '.md' || path.extname(file) == '.markdown' || path.extname(file) == '.mkdn' || path.extname(file) == '.mdown') {
```

Comment:
Me parece que estaría bien solo con `.md` y `.markdown`, los otros 2 no los he visto, dónde los has encontrado? 🤔


```suggestion
      // Esta línea podría ir en el top de tu función para que no sea recreada en el loop ya que nunca cambiará. 
      const supportedExtensions = ['.md', '.markdown']
      const extension = path.extname(file)

      if (supportedExtensions.includes(extension)) {
```